### ISSUE TITLE
how to use gc_loss

### ISSUE THREAD

**Issue by cs-xiao** [yuta-hi/pytorch_similarity#3]
Hi, I am confuse of how to correctly use the gc_loss.
for example:

other_loss = myModel(x)
model = GradientCorrelation2d(return_map=False)
gc_loss = 1. - model( input1, input2 )
loss = other_loss  + gc_loss 
optimizier.zero_grad()
loss .backward()
optimizier.step()

Should I need to  fixed the params of model or make the grad of model zero before every backward?

Thanks~
